hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/span/SpanMapManager.java

```diff
@@ -318,7 +318,7 @@ private void rationalizeImpliedTransfers(final TxnAccessor accessor) {
         }
     }
 
-    private void expandImpliedTransfers(final TxnAccessor accessor) {
+    public void expandImpliedTransfers(final TxnAccessor accessor) {
```

Comment:
Needs a comment saying what "expanding implied transfers" really means.  Especially now that it's public.

Reply:
And now that it's public a unit test would be appropriate but probably you think it unnecessary.

Plus argument `accessor` needs a `@NonNull`